static/app/views/explore/spans/spansTab.tsx

```diff
@@ -398,7 +398,8 @@ const Body = styled(Layout.Body)<{withToolbar: boolean}>`
     grid-template-rows: auto 1fr;
     align-content: start;
     gap: ${space(2)} ${p => (p.withToolbar ? `${space(2)}` : '0px')};
-    transition: 700ms;
+    will-change: grid-template;
+    transition: grid-template 200ms cubic-bezier(0.22, 1, 0.36, 1);
```

Comment:
Scope transition to `grid-template` property and make it faster

Reply:
to avoid getting different transitions everywhere, I think we should have pre-defined transitions on the `theme` as tokens that devs can use for certain situations. Or at least have the timings (200ms) in the theme?

Reply:
I agree! I will work on getting these defined in our theme.